MSAL/src/requests/MSALBaseRequest.m

```diff
@@ -167,7 +167,13 @@ - (void)acquireToken:(nonnull MSALCompletionBlock)completionBlock
     NSMutableDictionary<NSString *, NSString *> *reqParameters = [NSMutableDictionary new];
     
     // TODO: Remove once uid+utid work hits PROD
-    NSURLComponents *tokenEndpoint = [NSURLComponents componentsWithURL:_authority.tokenEndpoint resolvingAgainstBaseURL:NO];
+    NSURLComponents *tokenEndpoint = [NSURLComponents componentsWithURL:_authority.tokenEndpoint
```

Comment:
can this be an util (swapping host)?

Reply:
Removing as agreed - instance aware flow should be addressed separately.